src/LocalCoords.h

```diff
@@ -93,14 +99,16 @@ class LocalCoords {
   void setLattice(Lattice* lattice);
   void setLatticeX(int lattice_x);
   void setLatticeY(int lattice_y);
+  void setLatticeZ(int lattice_z);
   void setX(double x);
   void setY(double y);
+  void setZ(double z);
   void setNext(LocalCoords *next);
   void setPrev(LocalCoords* coords);
 
   LocalCoords* getLowestLevel();
   LocalCoords* getHighestLevel();
-  void adjustCoords(double delta_x, double delta_y);
+  void adjustCoords(double delta_x, double delta_y, double delta_z=0.0);
```

Comment:
Is `LocalCoords:adjustCoords(...)` used anywhere with the default `delta_z` argument? If not, I would remove the default value and require it to be set everywhere (or make all args have a default).